src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -163,6 +163,45 @@
       }
     }
   }
+  .cautionary-information-new {
```

Comment:
I think you can apply the changes needed for this to work directly to `.cautionary-information`.

Reply:
Once the prod flag is removed changes will be in `.cautionary-information`